tools/src/main/scala/scala/scalanative/interflow/Visit.scala

```diff
@@ -45,6 +46,10 @@ trait Visit { self: Interflow =>
         }
     }
 
+  /** In debug mode, visits all reachable definitions from defined entries. In
+   *  release mode, visits all entry symbols.
+   */
+  // QUESTION: What is exactly an entry?
```

Comment:
Entry is a root symbol from which we should start visting the methods. 
Typically it would be the main method symbol.
In case of using ScalaNative to create a library entry would be every method annotated with `@exported` found on the classpath.

There also exist a synthetic entries - for example a list of symbols that might not be directly reachable from the organic roots, but might be required in optimizer or in lowering stage (see `Lower.depends`, `Generate.depends`, `Interflow.depends`)